infra/charts/turing/README.md

```diff
@@ -80,15 +79,16 @@ The following table lists the configurable parameters of the Turing chart and th
 | mlp.postgresql | object | `{"nameOverride":"postgresql-mlp","postgresqlPassword":"mlp"}` | Postgresql configuration to be applied to MLP's postgresql database deployment Reference: https://artifacthub.io/packages/helm/bitnami/postgresql/8.9.8#parameters |
 | mlp.postgresql.nameOverride | string | `"postgresql-mlp"` | Name of MLP's Postgresql deployment |
 | mlp.postgresql.postgresqlPassword | string | `"mlp"` | Password for MLP Postgresql database |
-| postgresql | object | `{"metrics":{"enabled":false,"replication":{"applicationName":"turing","enabled":false,"numSynchronousReplicas":2,"password":"repl_password","slaveReplicas":2,"synchronousCommit":"on","user":"repl_user"},"serviceMonitor":{"enabled":false}},"persistence":{"enabled":true,"size":"10Gi"},"postgresqlDatabase":"turing","postgresqlPassword":"turing","postgresqlUsername":"turing","resources":{"requests":{"cpu":"500m","memory":"256Mi"}}}` | Postgresql configuration to be applied to Turing's postgresql database deployment Reference: https://artifacthub.io/packages/helm/bitnami/postgresql/8.9.8#parameters |
+| postgresql | object | `{"metrics":{"enabled":false,"replication":{"applicationName":"turing","enabled":false,"numSynchronousReplicas":2,"password":"repl_password","slaveReplicas":2,"synchronousCommit":"on","user":"repl_user"},"serviceMonitor":{"enabled":false}},"persistence":{"enabled":true,"size":"10Gi"},"postgresqlDatabase":"turing","postgresqlPassword":"turing","postgresqlPort":5432,"postgresqlUsername":"turing","resources":{"requests":{"cpu":"500m","memory":"256Mi"}}}` | Postgresql configuration to be applied to Turing's postgresql database deployment Reference: https://artifacthub.io/packages/helm/bitnami/postgresql/8.9.8#parameters |
 | postgresql.persistence.enabled | bool | `true` | Persist Postgresql data in a Persistent Volume Claim |
 | postgresql.postgresqlPassword | string | `"turing"` | Password for Turing Postgresql database |
 | postgresql.resources | object | `{"requests":{"cpu":"500m","memory":"256Mi"}}` | Resources requests and limits for Turing database. This should be set according to your cluster capacity and service level objectives. Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ |
 | sentry.dsn | string | `""` | Sentry DSN value used by both Turing API and Turing UI |
-| tags.db | bool | `true` | Specifies if Postgresql database needs to be installed together with Turing |
+| tags.db | bool | `true` |  |
```

Comment:
Do we want to describe what this config is for?

Reply:
I think @leonlnj last time manually added this into the `infra/charts/turing/README.md`. 

@leonlnj, this readme is generated by using [norwoodj/helm-docs](https://github.com/norwoodj/helm-docs) from the inline comments on the values.yaml file. Example:
https://github.com/gojek/turing/blob/335efc9464f5ed99bcf9d65973205f1f36ea019f/infra/charts/turing/values.yaml#L314

This can probably automated with git hooks

Reply:
Fixed it now. Thanks for catching, @terryyylim